pooch/core.py

```diff
@@ -440,12 +441,22 @@ def load_registry(self, fname):
 
         Parameters
         ----------
-        fname : str
-            File name and path to the registry file.
+        fname : str | fileobj
+            File path to the registry file, of file object.
```

Comment:
```suggestion
            Path (or open file object) to the registry file.
```

Reply:
This is a tricky phrasing. Does this make it less ambiguous?

Reply:
Ah yes, thanks